src/plugins/intel_cpu/thirdparty/CMakeLists.txt

```diff
@@ -132,3 +132,19 @@ function(ov_add_onednn)
 endfunction()
 
 ov_add_onednn()
+
+if(ENABLE_CPU_EXTENSIONS)
+    if(CMAKE_COMPILER_IS_GNUCC OR CMAKE_COMPILER_IS_GNUCXX)
+        if(NOT CMAKE_CXX_COMPILER_VERSION VERSION_LESS "11.2")
+            set(COMPILE_CPU_EXTENSIONS TRUE CACHE INTERNAL "")
+        endif()
+    elseif(OV_COMPILER_IS_CLANG)
+        if(NOT CMAKE_CXX_COMPILER_VERSION VERSION_LESS "12")
+            set(COMPILE_CPU_EXTENSIONS TRUE CACHE INTERNAL "")
+        endif()
+    endif()
+
+    if(COMPILE_CPU_EXTENSIONS)
+        add_subdirectory(cpu_extensions EXCLUDE_FROM_ALL)
```

Comment:
for static build case we need to install these libraries as well using `ov_install_static_lib` cmake function

Reply:
Done.